Combination-sum-leetcode-problem.py

```diff
@@ -0,0 +1,23 @@
+class Solution:
+    def combinationSum(self, candidates: List[int], target: int) -> List[List[int]]:
+        self.ans = []
```

Comment:
Instead of using class variable to store the result for the solution it would be better if you use local variable because using class variable may cause an issue when different object/instances of this class is created simultaneously.